mne/channels/channels.py

```diff
@@ -439,13 +439,13 @@ class UpdateChannelsMixin(object):
     def pick_types(self, meg=True, eeg=False, stim=False, eog=False,
                    ecg=False, emg=False, ref_meg='auto', misc=False,
                    resp=False, chpi=False, exci=False, ias=False, syst=False,
-                   seeg=False, bio=False, ecog=False, include=[],
-                   exclude='bads', selection=None):
+                   seeg=False, dipole=False, gof=False, bio=False, ecog=False,
+                   include=[], exclude='bads', selection=None):
```

Comment:
lol. We can raise an error "uh oh, we didn't deprecate because of botched up probability calculation but you're welcome to help us correct the calculation" :P I guess the number `1e-12` depends on the number of users we have ...